### PR TITLE
add svg width height property support

### PR DESCRIPTION
**Pull request by ndee02**
proposed implementation for https://github.com/sphinx-contrib/plantuml/issues/71

### CODE REVIEW

File: sphinxcontrib/plantuml.py [hunk 1]
```diff
@@ -442,13 +443,24 @@ def _get_svg_style(fname):
...
+def _svg_style_subst_width_height(style_str, wh_style_subst):
+    # remove width and height
+    style_str_wo_wh = re.sub(r'width:[^;]+[;]?|height:[^;]+[;]?', '', style_str)
+    return '%s; %s' % (style_str_wo_wh, wh_style_subst)
+
+def _svg_get_style_str(node, outfname):
+	style_str = _get_svg_style(outfname) or ''    
+	svg_attribs = ["%s:%s" % (key, val) for key, val in node.attributes.items() if key in ['width', 'height', 'max-width']]
+	return _svg_style_subst_width_height(style_str, '; '.join(svg_attribs))
+
```
[the remark this answers is not in the document]
Comment (ndee02) [reply sphinxcontrib/plantuml.py:455]: Agreed, I was just wondering about the purpose of copying the whole style of the SVG element was. If it is just to get width and height to set the dimensions of the `<object>` then I would also prefer your suggestion.

File: sphinxcontrib/plantuml.py [hunk 2]
```diff
@@ -443,15 +443,14 @@ def _get_svg_style(fname):
...
-    # remove width and height
-    style_str_wo_wh = re.sub(r'width:[^;]+[;]?|height:[^;]+[;]?', '', style_str)
-    return '%s; %s' % (style_str_wo_wh, wh_style_subst)
-
 def _svg_get_style_str(node, outfname):
-	style_str = _get_svg_style(outfname) or ''    
-	svg_attribs = ["%s:%s" % (key, val) for key, val in node.attributes.items() if key in ['width', 'height', 'max-width']]
-	return _svg_style_subst_width_height(style_str, '; '.join(svg_attribs))
+    width_height_styles = ["%s:%s" % (key, val) for key, val in node.attributes.items() if key in ['width', 'height', 'max-width']]
+    if len(width_height_styles) > 0:
```
Comment (yuja) [inline sphinxcontrib/plantuml.py:448]: Nit: you can simply write `if width_height_styles:`

**Review by yuja — APPROVED** [review]
New version mostly looks good to me, thanks.

File: sphinxcontrib/plantuml.py (hunk 2 above)
Comment (ndee02) [reply sphinxcontrib/plantuml.py]: Agreed. Seems more idiomatic.

**Pull request closed without merging**